pkgs/by-name/libresoc/power-instruction-analyzer.nix

```
@@ -0,0 +1,28 @@
+{
+  python39Packages,
+  rustPlatform,
+  fetchFromLibresoc,
+}:
+with python39Packages;
+with rustPlatform;
+  buildPythonPackage rec {
```

```suggestion
  python39Packages.buildPythonPackage rec {
```

Please limit the scope of `with`. I believe you only use `python39Packages` to call `buildPythonPackage`? If that is really true, then just use it without `with`.